lib/web_ui/lib/src/engine/layers.dart

```diff
@@ -317,13 +293,7 @@ class TransformOperation implements LayerOperation {
   Matrix4 getMatrix() => Matrix4.fromFloat32List(toMatrix32(transform));
 
   @override
-  ui.Rect cullRect(ui.Rect contentRect) => getMatrix().transformRect(contentRect);
-
-  @override
-  ui.Rect inverseMapRect(ui.Rect rect) {
-    final Matrix4 matrix = getMatrix()..invert();
-    return matrix.transformRect(rect);
-  }
+  ui.Rect mapRect(ui.Rect contentRect) => getMatrix().transformRect(contentRect);
```

Comment:
Is the matrix immutable? If so, can we create the matrix in the constructor and reuse it when rendering?